apps/sms/views/conversation/js/conversation.js

```diff
@@ -1923,6 +1968,21 @@ var ConversationView = {
     return this.inEditMode;
   },
 
+  addRecipients: function thui_add_recipients() {
+
+    // redirect to Composer panel and add all the contacts
+    return Navigation.toPanel('composer', {
+      composer: {
+        subject: Compose.getSubject(),
+        type: Compose.type,
+        body: Compose.getText(),
+        content: Compose.getContent()
+      },
+      recipients: Threads.active.participants
```

Comment:
because we use URLs now, we can't use complex objects anymore, so instead we'll need to save as a draft and pass the draftId (look at initiateNewMessage for more information)